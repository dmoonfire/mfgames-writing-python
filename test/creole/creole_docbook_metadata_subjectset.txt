= Document Heading
* Subject1: This; Is; Four; Terms
* Subject2: Only; Two

The above document only has two subjects.
