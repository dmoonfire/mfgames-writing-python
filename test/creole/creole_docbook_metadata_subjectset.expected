<?xml version="1.0" encoding="UTF-8"?><article xmlns='http://docbook.org/ns/docbook' version="5.0"><info><title>Document Heading</title><subjectset schema="Subject1"><subject><subjectterm>This</subjectterm></subject><subject><subjectterm>Is</subjectterm></subject><subject><subjectterm>Four</subjectterm></subject><subject><subjectterm>Terms</subjectterm></subject></subjectset><subjectset schema="Subject2"><subject><subjectterm>Only</subjectterm></subject><subject><subjectterm>Two</subjectterm></subject></subjectset></info><simpara>The above document only has two subjects.</simpara></article>
